frontend/clients/mentorship.js

```diff
@@ -1,3 +1,3 @@
 import { create } from "axios";
 
-export const mentorshipAPI = create({ baseURL: process.env.url });
+export const mentorshipAPI = create({ baseURL: 'http://localhost:1337' });
```

Comment:
- we get the values of environmental variables from `nextjs.config.js`. if you look at that file you will find the below line of code.

>   env: {
    url: process.env.M3NTORSHIP_APP_STRAPI_URL,
  },
the value of `M3NTORSHIP_APP_STRAPI_URL` should be stored in your OS environamntal variables. 
[next js environmental variables](https://nextjs.org/docs/api-reference/next.config.js/environment-variables)